conda-recipe/run_test.sh

```diff
@@ -8,6 +8,6 @@ source ${ONEAPI_ROOT}/tbb/latest/env/vars.sh
 
 set -x
 
-pytest -q -ra --disable-warnings --pyargs numba_dppy -vv
+pytest -q -ra --disable-warnings --cov numba_dppy --cov-report term-missing --pyargs numba_dppy -vv
```

Comment:
Please, try to use config file for these arguments.

Reply:
Type of report cannot be specified in [.coveragerc](https://stackoverflow.com/a/30754081/6597765)